src/DeckSurf.SDK/Core/DeviceManager.cs

```diff
@@ -17,6 +17,7 @@ namespace DeckSurf.SDK.Core
     public class DeviceManager
     {
         private static readonly int SupportedVid = 4057;
+        private static readonly int SupportedVidOriginalV2 = 6940;
```

Comment:
Was the VID actually different for the original? I also am swapping this to an array (see #4) to make it easier to parse out.